tests/e2e/storage.e2e-spec.ts

```diff
@@ -0,0 +1,420 @@
+require('dotenv').config();
+import request from 'supertest';
+import { HttpStatus } from '@nestjs/common';
+import { encryptFilename, createTestUser, deleteTestUser } from './utils';
+import { Sign } from '../../src/app/middleware/passport';
+import { applicationInitialization } from './setup';
+import { FileModel } from '../../src/app/models/file';
+import { FolderModel } from '../../src/app/models/folder';
+import sequelize from 'sequelize';
+const { Op } = sequelize;
+const server = require('../../src/app');
+const app = server.express;
+
+const deleteAllUserFilesFromDatabase = async (userId: number): Promise<void> => {
+  const files = await server.models.file.findAll({ where: { user_id: userId } });
+  await files.forEach((file: FileModel) => file.destroy());
```

Comment:
I'm not sure you can do await on forEach [1]

perhaps doing a 

for(const file of files){
 await file.destroy()
}

might work?

[1]: https://stackoverflow.com/a/37576787/4619199